docs/source-control/new-repo.md

```diff
@@ -0,0 +1,46 @@
+---
+id: new-repo
+slug: /source-control/new-repo
+title: New Repository
+
+---
+
+## Basic Setup
+
+* Initialize the repository with `git init` (cli)
+* Create a new repository on github (github)
+* Select The Art of Education as the owner (github)
```

Comment:
🤔 The "how" of doing this is a little unclear. I found [this](https://docs.github.com/en/organizations/managing-peoples-access-to-your-organization-with-roles/maintaining-ownership-continuity-for-your-organization) while googling - are these valid steps to follow? 

I did notice when creating my repo that it was created under The Art of Education but wasn't sure if that completed the task of making "The Art of Education" as the owner.

Reply:
The name of the owner is a drop-down and you can switch between you and any organization you belong to